apps/nextjs/src/app/quiz-designer/page.tsx

```diff
@@ -1,5 +1,15 @@
+import { redirect } from "next/navigation";
+
+import { serverSideFeatureFlag } from "@/utils/serverSideFeatureFlag";
+
 import QuizDesignerPage from "./quiz-designer-page";
 
-export default function QuizDesigner() {
+export default async function QuizDesigner() {
+  const canSeeQuizDesigner = await serverSideFeatureFlag("show-qd");
```

Comment:
`quiz-designer` might be a clearer name, as the feature flag kind of implies that it's about showing/hiding something